src/wavbreaker.c

```diff
@@ -3600,3 +3562,38 @@ int track_breaks_load_from_file( char* filename) {
     force_redraw();
     return 0;
 }
+
+/** @param str Time in MM:SS:FF format (where there are CD_BLOCKS_PER_SEC frames per second).
+ *  @return offset in frames.
+ */
+guint
+msf_time_to_offset( gchar *str )
+{
+    char   buf[1024];
+    uint   parse[3];
+    gchar  *aptr, *bptr;
+    int    index;
+    uint   offset;
+
+
+    strncpy( buf, str, 1024 );
+
+    aptr = buf;
+    for( index = 0; index < 2; index++ ) {
```

Comment:
Looks like this code was already like this before, but now it makes me wonder if we should just replace this with, especially since the code has some other defects like reading out of bounds with `memchr()` when aptr > buf and there is no `:` in the remainder of the string:

```
int mm=0, ss=0, ff=0;
int consumed = sscanf(str, "%d:%d:%d", mm, ss, ff);
if (consumed == 3) {
    ...
}
```